docs/book/unit-testing.md

```diff
@@ -364,27 +409,27 @@ protected $albumTable;
 Next, we'll create three new methods that we'll invoke during setup:
 
 ```php
-protected function configureServiceManager(ServiceManager $services)
-{
-    $services->setAllowOverride(true);
+protected function configureServiceManager(ServiceManager $services): void
+    {
+        $services->setAllowOverride(true);
```

Comment:
The indentation is wrong, the 4 spaces at the beginning are not necessary.

Reply:
Corrected